src/database/schema.sql

```diff
@@ -49,4 +49,20 @@ CREATE TABLE books (
   publisher VARCHAR(200) NOT NULL,
   cover_state VARCHAR(4) NOT NULL,
   PRIMARY KEY(id)
+);
+
+-- Music Album class table
+CREATE TABLE music_album (
+  id SERIAL PRIMARY KEY,
+  item_id int FOREIGN KEY REFERENCES item(id),
```

Comment:
As per the instructions, the child class tables should have ALL of the properties (label, author, genre) from the parent Item table, which then reference the association tables. The item_id  attribute is not necessary

Reply:
Thaank you @lucascomamala for this review comment.Let me implement it